### ISSUE TITLE
Arduino Yún: Error 107 "invalid JSON" when repeatedly saving an object

### ISSUE THREAD

**Issue by pascalgiguere** [ParsePlatform/parse-embedded-sdks#44]
I have a very basic Arduino Yún sketch which only saves a new object with a hardcoded key-value pair every minute.

```c
void loop() {
  ParseObjectCreate create;
  create.setClassName("TestObject");
  create.add("testKey", 27.6);

  ParseResponse response = create.send();
  Serial.print(response.getJSONBody());
  response.close();

  delay(60000);
}
```

This will successfully save the object initially.

However if I keep the sketch running for a while (~30 minutes), I will start getting the error `{"code":107,"error":"invalid JSON"}` on every save.

Restarting the sketch will make saves work again, until I wait ~30 minutes and it stops working again.

**Comment by mtl2034** [comment]
Hi, I suspect this happens because of memory leaks. What version of the library do you use? We fixed number of memory leaks back in April and we had some performance fixes recently.
If you are not using 1.0.1-rc3 can you try it?

**Comment by pascalgiguere** [comment]
I was using version 1.0.0 (which is currently linked in Parse.com's guides).
It does seem like a memory leak. I will try 1.0.1-rc3 and report back.

**Comment by pascalgiguere** [comment]
7 hours running and no error anymore, so the issue seems resolved.
Thanks for the quick reply!

**Comment by mtl2034** [comment]
Glad to hear. I am going close this issue then. If you encounter it again please reopen.

**Comment by abhirocks550** [comment]
How can i update my library to 1.0.1-rc3. Can you please help? @mtl2034

**Comment by mtl2034** [comment]
@abhirocks550, what platform do you use? Please note we recently released version 1.0.1
